lib/internal/source_map/source_map_cache.js

```diff
@@ -195,11 +190,9 @@ function rekeySourceMap(cjsModuleInstance, newInstance) {
 function sourceMapCacheToObject() {
   const obj = ObjectCreate(null);
 
-  const it = MapPrototypeEntries(esmSourceMapCache);
-  let entry;
-  while (!(entry = MapIteratorNext(it)).done) {
-    const k = entry.value[0];
-    const v = entry.value[1];
+  for (const entry of esmSourceMapCache) {
+    const k = entry[0];
+    const v = entry[1];
```

Comment:
This can use object (`[[Get]]`) destructuring, which is equivalent to the current code:
```suggestion
  for (const { 0: k, 1: v } of esmSourceMapCache) {
```